krabs/krabs/filtering/event_filter.hpp

```diff
@@ -62,6 +71,11 @@ namespace krabs {
         template <typename U>
         void add_on_event_callback(const U &callback);
 
+		unsigned short OrigEventId()const
+		{
+			return m_OrigEventId;
```

Comment:
nit: `providerFilterEventId` here and elsewhere

Reply:
`provider_filter_event_id`.